databases/spatialite-tools/Portfile

```diff
@@ -3,35 +3,36 @@
 PortSystem 1.0
 
 name                    spatialite-tools
-version                 5.0.0
+version                 5.1.0
 revision                0
 categories              databases gis
 license                 GPL-3
 platforms               darwin
-maintainers             {yahoo.com:n_larsson @nilason} open maintainer
+maintainers             {yahoo.com:n_larsson @nilason} openmaintainer
 description             Variety of CLI tools for SpatiaLite DB
 long_description        These CLI tools allow to interact with a SpatiaLite DB,\
                         import a variety of files (SHP, OSM, GML) and\
                         prepare a virtual network for routing.
 
-homepage                https://www.gaia-gis.it/fossil/spatialite-tools/index
-master_sites            https://www.gaia-gis.it/gaia-sins/
-distname                spatialite-tools-${version}
+homepage                https://www.gaia-gis.it/fossil/${name}/index
+master_sites            https://www.gaia-gis.it/gaia-sins/${name}-sources
+distname                ${name}-${version}
```

Comment:
is this required, should be the default right?

Reply:
default is good, removed